### PR TITLE
added two files

### PR DESCRIPTION
**Pull request by lizaaadrozd**

### CODE REVIEW

File: liza/array_values_check.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+def check_values_greater5(input_array)
```
Comment (NickOvc) [inline liza/array_values_check.rb:3]: `greater_than_five` is more appropriate method name

File: liza/array_values_check.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  amount_items_greater5 = 0
```
Comment (NickOvc) [inline liza/array_values_check.rb:4]: `greater_than_five_items_amount`

File: liza/array_values_check.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```
Comment (NickOvc) [inline liza/array_values_check.rb:1]: what does this do?

**Review by NickOvc — APPROVED** [review]

File: liza/array_values_check.rb (hunk 1 above)
Comment (lizaaadrozd) [reply liza/array_values_check.rb]: Done

File: liza/array_values_check.rb (hunk 2 above)
Comment (lizaaadrozd) [reply liza/array_values_check.rb]: Done

File: liza/array_values_check.rb (hunk 3 above)
Comment (lizaaadrozd) [reply liza/array_values_check.rb]: Deleted frozen string

File: liza/number_check.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+def number_check(num)
+  result = if qualify?(num)
+             'The number is qualify.'
+           else
+             'The number is not qualify.'
```
Comment (NickOvc) [inline liza/number_check.rb:5]: The number qualifies
The number does not qualify

**Review by NickOvc — APPROVED** [review]

File: liza/number_check.rb (hunk 4 above)
Comment (lizaaadrozd) [reply liza/number_check.rb]: Done

**Pull request merged**